ztp/siteconfig-generator-kustomize-plugin/testSiteConfig/site1-sno-du.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   clusters:
   - clusterName: "site-sno-du-1"
     biosConfigRef:
-      filePath: "testSiteConfig/testHW.profile"
+      filePath: "../siteconfig-generator-kustomize-plugin/testSiteConfig/testHW.profile"
```

Comment:
why did you need to change that ?

Reply:
I saw there was an error while executing 
```
siteconfig-generator]$  ./siteconfig-generator -manifestPath ../source-crs/extra-manifest ../siteconfig-generator-kustomize-plugin/testSiteConfig/site2-standard-du.yaml
2023/06/12 13:56:33 Error: could not build the entire SiteConfig defined by ../siteconfig-generator-kustomize-plugin/testSiteConfig/site2-standard-du.yaml: open testSiteConfig/testHW.profile: no such file or directory
```
because the testHW.profile didn't exist at the specified filePath and thought to address this.


I can remove this change, since it's outside the scope of this PR.
@tliu2021, what do you think?

Reply:
The file path mentioned here is relative to the kustomization.yaml file, and this test file is intended to be utilized by the Makefile within the siteconfig-generator-kustomize-plugin. If you wish to use this test file outside of the siteconfig-generator-kustomize-plugin,  it may be beneficial to use the proposed path.

Reply:
My reasoning was that since the [siteconfig-generator README](https://github.com/openshift-kni/cnf-features-deploy/tree/master/ztp/siteconfig-generator) suggests using the SiteConfig files from siteconfig-generator-kustomize-plugin/testSiteConfig, then all of them should successfully run as  siteconfig-generator parameters.

Reply:
I think your proposed change looks good.

Reply:
Thanks, @tliu2021! We've decided to remove this change from this PR as it's unrelated. Might make a new PR with this small change in the future.